### DISCUSSION TITLE
distributed-transaction/CAP

### DISCUSSION THREAD

**Discussion by giscus** [isno/theByteBook#153] [General]
# distributed-transaction/CAP

构建大规模高可用的分布式系统

https://www.thebyte.com.cn/distributed-transaction/CAP.html

**Comment by lbsu** [comment]
"而不是三选二而的权衡。" 
多了个"而"？

**Reply by isno** [reply]
谢谢，已更正。

**Comment by supervons** [comment]
> CAP 是分不是系统中关于一致性与可用性的权衡理论
分布式

**Reply by isno** [reply]
谢谢，已修正。

**Comment by likz1994** [comment]
文中“将 CAP、ACID 中讨论的一致性（P）称为强一致性” ，应该是 “将 CAP、ACID 中讨论的一致性（C）称为强一致性”吧？

**Reply by isno** [reply]
谢谢，这里是我写错了
